src/cpp/session/resources/dependencies/r-packages.json

```diff
@@ -441,27 +390,20 @@
         "shiny": {
             "description": "Shiny",
             "packages": [
-                "R6",
-                "bslib",
-                "cachem",
-                "commonmark",
-                "crayon",
-                "digest",
-                "ellipsis", 
-                "fastmap",
-                "glue",
-                "htmltools",
+                "Rcpp",
                 "httpuv",
-                "jsonlite",
-                "later",
-                "lifecycle",
                 "mime",
+                "jsonlite",
+                "xtable",
```

Comment:
nit: can we keep these alphabetically sorted? (easier to can through + makes diffs a little cleaner)